src/components/08-navigation/nav-primary.njk

```diff
@@ -1,5 +1,5 @@
 <button class="usa-nav-close">
-  <img src="{{ uswds.path }}/img/close.svg" alt="close">
+  <img src="{{ uswds.path }}/img/close-primary.svg" alt="close">
```

Comment:
This doesn't seem to be working 
![screen shot 2017-08-31 at 11 31 05 am](https://user-images.githubusercontent.com/776987/29934207-ecda36c8-8e3f-11e7-9cfe-e5e66c7ba4b7.png)

Reply:
On which one of the nav pages are you seeing the button not appearing? I had this problem locally at first, and I ended up having to run `npm run build` every time I changed an image, otherwise they wouldn't be copied over to the dist folders correctly. I'm seeing this on the Federalist for the basic nav:

![screen shot 2017-08-31 at 11 15 34 am](https://user-images.githubusercontent.com/1701077/29938298-2b07cd50-8e3d-11e7-8e7e-c5cd7b67352f.png)

Reply:
hmmm it's magically working now